tools/mo/openvino/tools/mo/utils/cli_parser.py

```diff
@@ -471,11 +464,12 @@ def get_common_cli_options(model_name):
     d['scale'] = ['- Scale factor', lambda x: x if x else 'Not specified']
     d['data_type'] = ['- Precision of IR', lambda x: 'FP32' if x == 'float' else 'FP16' if x == 'half' else x]
     d['disable_fusing'] = ['- Enable fusing', lambda x: not x]
-    d['disable_gfusing'] = ['- Enable grouped convolutions fusing', lambda x: not x]
-    d['move_to_preprocess'] = '- Move mean values to preprocess section'
+    d['transform'] = ['- User transformations', lambda x: x if x else 'Not specified']
     d['reverse_input_channels'] = '- Reverse input channels'
-    d['use_legacy_frontend'] = '- Use legacy API for model processing'
+    d['static_shape'] = '- Enable IR generation for fixed input shape'
     d['transformations_config'] = '- Use the transformations config file'
+    d['use_legacy_frontend'] = '- Force the usage of legacy API for model processing'
```

Comment:
I afraid that term `legacy API` can be mixed with OV old API, but it is a different term. Can we provide more detailed explanation and separate it to a kind of advanced options? The same is about the next line.

Reply:
Done.